maintainers/maintainer-list.nix

```diff
@@ -14037,4 +14037,10 @@
     github = "nigelgbanks";
     githubId = 487373;
   };
+  jdupak = {
```

Comment:
According to the comment at the beginning of the file, this list of maintainers should be sorted alphabetically. I see that there are a few exceptions at the end of the file, but the rest is sorted.

Reply:
fixed